create_tables.py

```diff
@@ -1,29 +1,38 @@
+# -- coding: utf-8 --
+"""This script create the Sparkify database and all its tables. """
 import psycopg2
 from sql_queries import create_table_queries, drop_table_queries
+import argparse
 
 
-def create_database():
+def create_database(user_name: str, password: str):
     """
     - Creates and connects to the sparkifydb
     - Returns the connection and cursor to sparkifydb
     """
     
     # connect to default database
-    conn = psycopg2.connect("host=127.0.0.1 dbname=studentdb user=student password=student")
+    conn = psycopg2.connect(
```

Comment:
Great work. Could you add context manager when connecting to de DDBB? Otherwise, we could left open the database connection when if something goes wrong during the function call.

Reply:
Hi Heber,  I couldn´t implement the context manager gestor. However, the exception was made with a try.
let me know if you have any questions.

Reply:
Ok